services/src/main/java/org/exoplatform/addons/gamification/storage/dao/GamificationHistoryDAO.java

```diff
@@ -34,11 +34,14 @@
 import org.exoplatform.addons.gamification.IdentityType;
 import org.exoplatform.addons.gamification.entities.domain.effective.GamificationActionsHistory;
 import org.exoplatform.addons.gamification.service.dto.configuration.RealizationsFilter;
+import org.exoplatform.addons.gamification.service.dto.configuration.constant.EntityType;
 import org.exoplatform.addons.gamification.service.dto.configuration.constant.HistoryStatus;
 import org.exoplatform.addons.gamification.service.effective.PiechartLeaderboard;
 import org.exoplatform.addons.gamification.service.effective.ProfileReputation;
 import org.exoplatform.addons.gamification.service.effective.StandardLeaderboard;
 import org.exoplatform.commons.persistence.impl.GenericDAOJPAImpl;
+import org.joda.time.DateTimeConstants;
+import org.joda.time.LocalDate;
```

Comment:
Should we use java.time instead of joda.time ?
As explained here https://github.com/JodaOrg/joda-time, jodatime library was previously used in before java 8.

> Joda-time is no longer in active development except to keep timezone data up to date. From Java SE 8 onwards, users are asked to migrate to java.time (JSR-310) - a core part of the JDK which replaces this project.

Reply:
You are right, I wasn't aware of `LocalDate` that was used from Joda time instead of Java Time. Thanks.
In fact, we have to make the cleanup to delete this from dependencies even.

fixed.